2022/incoming/looking-at-the-christmas-numbers.pod

```diff
@@ -0,0 +1,151 @@
+Author: lichtkind@cpan.org
+Title: Looking at the Christmas Numbers
+Topic: charts, colors
+
+=encoding utf8
+
+=head1 Looking at the Christmas Numbers
+
+Yes we want to put a heavinly smile on faces of people we treasure.
+But the expenses need to be kept in check - so lets create some bar charts.
+
+=head2 Grabbig some code
+
+The fastest start we get by copying some example from L<Chart::Manual::Types>, 
+where all details are explained. Then we change the first dataset 
+(animal names => gift recipients), and paste into the following sets our numbers.
+At last we label axis and legend and a first draft is ready:
+
+=begin perl
+
+    use Chart::Bars;
+    my $g = Chart::Bars->new( 600, 600 );
+    $g->add_dataset( qw/ Peter Paul Mary/ );
+    $g->add_dataset( 30, 40, 80 );
+    $g->add_dataset( 80, 60, 30 );
+    $g->add_dataset( 50, 30, 60 );
+    $g->set(
+        title         => 'Christmas Expenses',
+        x_label       => 'Recipient',
+        y_label       => 'Amount',
+        legend_labels => ['2020', '2021', '2022'],    
+        y_grid_lines  => 'true',
+        min_val       =>  0,
+        precision     =>  0,
+        colors => {
+            y_grid_lines => 'gray70',
+            misc         => 'gray55',
+            text         => 'gray55',
+            x_label      => 'gray40',
+            y_label      => 'gray40',
+            title        => 'gray20',
+        },
+    );
+    $g->png("bars1.png");
+
+=end perl
+
```

Comment:
Put a sentence here

Reply:
put there: The values under the key C<colors> just gray out the less important parts,
so that the chart is more pleasant.